docs/codesandbox/get-started-ajax/src/browser/app.jsx

```diff
@@ -1,7 +1,7 @@
-import * as hyperdom from "hyperdom";
-import styles from "./styles.css";
+const hyperdom = require("hyperdom");
```

Comment:
why require not import?

Reply:
People have an option to follow this Quick Start on their machine. The project is generated by `create-hyperdom-app`. It also includes electron based tests. But electron does not support es6 modules (because node does not support them) and so tests won't compile if the project is using `import`s.

I could add a babel transformation to tests, but that would slow them down, so I haven't.